GetIntoTeachingApi/Models/Crm/Candidate.cs

```diff
@@ -301,9 +301,10 @@ public enum RegistrationStatus
         [SwaggerSchema("Set to update the accepted privacy policy.", WriteOnly = true)]
         [EntityRelationship("dfe_contact_dfe_candidateprivacypolicy_Candidate", typeof(CandidatePrivacyPolicy))]
         public CandidatePrivacyPolicy PrivacyPolicy { get; set; }
+        [EntityRelationship("dfe_candidateschoolexperience", typeof(CandidateSchoolExperience))]
+        public List<CandidateSchoolExperience> SchoolExperiences { get; set; } = new List<CandidateSchoolExperience>();
 
         public Candidate()
-            : base()
```

Comment:
What's the reasoning behind this change? the `BaseModel` constructor has a purpose looking at this, so just want to make sure its intentional?

https://github.com/DFE-Digital/get-into-teaching-api/blob/master/GetIntoTeachingApi/Models/Crm/BaseModel.cs#L36

Reply:
Just following Visual Studio, did some reading and seems like you don't need that call to Base unless you are passing arguments. I'll put them back in to be consistent though
![image](https://user-images.githubusercontent.com/47089130/145058664-d22b33ac-f153-4e3c-aae6-1920b43f6768.png)